front/components/assistant/conversation/AgentSuggestion.tsx

```diff
@@ -0,0 +1,167 @@
+import { Button, DropdownMenu, RobotIcon } from "@dust-tt/sparkle";
+
+import { useAgentConfigurations } from "@app/lib/swr";
+import { AgentConfigurationType } from "@app/types/assistant/agent";
+import {
+  ConversationType,
+  isAgentMention,
+  isUserMessageType,
+  UserMessageType,
+} from "@app/types/assistant/conversation";
+import { WorkspaceType } from "@app/types/user";
+
+export function AgentSuggestion({
+  owner,
+  userMessage,
+  conversation,
+}: {
+  owner: WorkspaceType;
+  userMessage: UserMessageType;
+  conversation: ConversationType;
+}) {
+  const { agentConfigurations } = useAgentConfigurations({
+    workspaceId: owner.sId,
+  });
+  agentConfigurations.sort((a, b) => compareAgentSuggestions(a, b));
+
+  return (
+    <div className="pt-4">
+      <div className="text-xs font-bold text-element-600">
+        Which KillerZorg would you like to talk with?
+      </div>
+      <div className="mt-3 flex items-center gap-2">
+        {agentConfigurations.slice(0, 3).map((agent) => (
+          <Button
+            key={`message-${userMessage.sId}-suggestion-${agent.sId}`}
+            size="xs"
+            variant="avatar"
+            label={`@${agent.name}`}
+            onClick={() => selectSuggestionHandler(agent)}
+            avatar={agent.pictureUrl}
+          />
+        ))}
+        <DropdownMenu>
+          <DropdownMenu.Button>
+            <Button
+              variant="tertiary"
+              size="xs"
+              icon={RobotIcon}
+              label="Select another"
+            />
+          </DropdownMenu.Button>
+          <div className="relative bottom-6 z-30">
+            <DropdownMenu.Items origin="topLeft" width={320}>
```

Comment:
We should align width with the InputBar dropwdown menu (none there)

320 feels a bit large but maybe we can find a middle ground and align the two?

cc @Duncid

Reply:
Ok, should we aim at a single component used by both?

Reply:
Nope it's just a dropdown for now we just need to align the width

Reply:
Right now at least, but there's a task coming up that will make it a shared component

Reply:
IMO single component would be better (especially once we add search and other functions).
With 280 should be good.

Reply:
Fine to add it now or wait for that task